### PR TITLE
DDotDDD

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
Infrastructure "Creates DDotDDD, the dark days of the dead design document and add some basic region settings to the mod"

#### Purpose of change
I've been forsaking DDotD. 

#### Describe the solution
This establishes what I want to do with it, in the hopes of getting other interested parties.

As part of the initial push to create a design doc, I have also updated region settings somewhat, removing more of the non-classic elements from spawning at all and mildly adjusting some of the biome information to recreate alberta. I have, per the design doc, globally slowed down and toughened our non-reviving zombies.

Additionally although it has nothing to do with DDotD I added a critical missing bit of documentation to modding.md about the `monster_adjustment` field.

#### Describe alternatives you've considered
Magically doing everything I want in one go.

#### Testing
Loaded in game without errors. It's easier to run from zombies but harder to take them down. They're not really much harder proportionally as a result. The map looks a lot more like alberta. Good times.
image

#### Additional Context
Although it doesn't work, I added JSON to change the town and world names to Alberta versions.

### CODE REVIEW

File: data/mods/classic_zombies/design-doc.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### Zombification Mechanics
+The zombie affliction is infectious, possibly viral or fungal. Survivors may never find out. It comes in two forms: it transmits by airborn spores, which have a low infection rate and a long latency. After infection, carriers are very quickly able to infect others, but do not develop symptoms for 2-3 weeks. Once symptomatic it progresses quickly, with death usually occuring in another week. The infectious symptoms are vague and flu-like: fever, chills, severe nausea and vomiting, muscle aches, muscle spasms, and eventually delirium and death. Due to the relatively low infection rate of the airborn form, only around half of the population were susceptible to this form, and all survivors are clearly among that group. However, the much higher dose of infectious agent shared by bite or even scratch from a reanimated zombie carries an increased risk of infection and generates a much faster progressing form of the illness.
```
Comment (anothersimulacrum) [inline data/mods/classic_zombies/design-doc.md:31]: > Due to the relatively low infection rate of the airborn form, only around half of the population were susceptible to this form, and all survivors are clearly among that group.
The group that is susceptible? I am not sure this makes sense.
```suggestion
The zombie affliction is infectious, possibly viral or fungal. Survivors may never find out. It comes in two forms: it transmits by airborn spores, which have a low infection rate and a long latency. After infection, carriers are very quickly able to infect others, but do not develop symptoms for 2-3 weeks. Once symptomatic it progresses quickly, with death usually occurring in another week. The infectious symptoms are vague and flu-like: fever, chills, severe nausea and vomiting, muscle aches, muscle spasms, and eventually delirium and death. Due to the relatively low infection rate of the airborne form, only around half of the population were susceptible to this form, and all survivors are clearly among that group. However, the much higher dose of infectious agent shared by bite or even scratch from a reanimated zombie carries an increased risk of infection and generates a much faster progressing form of the illness.
```

File: data/mods/classic_zombies/design-doc.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+#### The Central Alberta Historical Re-Enactors Army
+CAHREA was a local club of creative anachronists. By some coincidence, a large number of their members were immune to the initial zombie infection. Their access to armour that is additionally protective against zombie bites gave them an edge, and they began helping evacuate and protect survivors in a rural community center. Although the original CAHREA group is now a small minority, they have maintained the name.
+
+#### The Wild Rose Survivors
+This is a group of rural Albertans less hard-hit by the pandemic portion of the plague due to their remote and spaced-out living. They were still eventually affected as the plague became widespread. They tend to be well equipped with pickup trucks and hunting rifles, somewhat insular, but welcoming - if a bit suspicoius - of other survivors.
```
Comment (actual-nh) [inline data/mods/classic_zombies/design-doc.md:50]:
```suggestion
This is a group of rural Albertans less hard-hit by the pandemic portion of the plague due to their remote and spaced-out living. They were still eventually affected as the plague became widespread. They tend to be well equipped with pickup trucks and hunting rifles, somewhat insular, but welcoming - if a bit suspicious - of other survivors.
```

**Comment by LyleSY** [comment]
Alberta is also one of the best places to find dino fossils in the world. Only a few of the dinos found there are in current DinoMod https://en.wikipedia.org/wiki/Dinosaur_Provincial_Park

File: data/mods/classic_zombies/design-doc.md (hunk 1 above)
Comment (actual-nh) [reply data/mods/classic_zombies/design-doc.md]: I would likewise think the survivors would be clearly NOT in the susceptible group.

**Comment by I-am-Erk** [comment]
> 
> 
> Alberta is also one of the best places to find dino fossils in the world. Only a few of the dinos found there are in current DinoMod https://en.wikipedia.org/wiki/Dinosaur_Provincial_Park

Sure is! The alberta badlands are a really interesting place to visit.

File: data/mods/classic_zombies/design-doc.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+The first wave of zombies further spread the illness due to unreadiness and the population already being overwhelmed by the new pandemic. The death toll was catastrophic, particularly among medical personnel caring for those infected with the illness. It isn't known exactly what went wrong with military efforts, but they were undoubtedly stymied by their own ranks falling ill will the airborne form of the pandemic even as they tried to fight the zombies.
```
Comment (actual-nh) [inline data/mods/classic_zombies/design-doc.md:33]:
```suggestion
The first wave of zombies further spread the illness due to unreadiness and the population already being overwhelmed by the new pandemic. The death toll was catastrophic, particularly among medical personnel caring for those infected with the illness. It isn't known exactly what went wrong with military efforts, but they were undoubtedly stymied by their own ranks falling ill with the airborne form of the pandemic even as they tried to fight the zombies.
```

File: data/mods/classic_zombies/design-doc.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Four months ago, new reports of the same effect came up almost simultaneously in Morocco, Belarus, Vietnam, and Portugal. The WHO expressed concern and gave advice for avoiding wild animals. Within a month, the plague had spread to most of the planet. The initial caution about wild animals proved a dangerous red herring, causing people to cluster in urban centers, where the actual vector - other humans and some infected vermin - spread the zombie affliction rapidly. Unfortunately the long latency of the agent meant that it was another month before the ubiquity of the infection became apparent. By the time the second wave of infections became clear, millions of people had been infected, and already passed it to the currently unaware third wave. Hospitals were rapidly overwhelmed, before the revivification risk was even clear. Critically, police and military were among those affected by the unrecognized first wave, and their ranks were as decimated as civilians.
```
Comment (actual-nh) [inline data/mods/classic_zombies/design-doc.md:31]:
```suggestion
Four months ago, new reports of the same effect came up almost simultaneously in Morocco, Belarus, Vietnam, and Portugal. The WHO expressed concern and gave advice for avoiding wild animals. Within a month, the plague had spread to most of the planet. The initial caution about wild animals proved a dangerous red herring, causing people to cluster in urban centers, where the actual vector - other humans and some infected vermin - spread the zombie affliction rapidly. Unfortunately the long latency of the agent meant that it was another month before the ubiquity of the infection became apparent. By the time the second wave of infections became clear, millions of people had been infected, and already passed it to the currently unaware third wave. Hospitals were rapidly overwhelmed, before the revivification risk was even clear. Critically, police and military were among those affected by the unrecognized first and second waves, and their ranks were as decimated as civilians.
```

**Review by actual-nh — COMMENTED** [review]
Interesting!

File: data/mods/classic_zombies/modinfo.json [hunk 5]
```diff
@@ -3,15 +3,21 @@
     "type": "MOD_INFO",
     "id": "classic_zombies",
     "name": "Dark Days of the Dead",
-    "authors": [ "Hirmuolio" ],
+    "authors": [ "Hirmuolio", "I-am-Erk" ],
     "maintainers": [ "I-am-Erk" ],
     "description": "(Formerly Classic Zombies): Only spawn classic zombies (normal zombies and zombie animals) and natural wildlife.  Zombies do not revive and do not need pulping.  This disables certain buildings and map extras.  The long term plan is to make this a classic zombie movie simulator set in the present day.  If you have this mod enabled in experimental, some items may disappear.",
-    "category": "monster_exclude",
+    "category": "content",
```
Comment (anothersimulacrum) [inline data/mods/classic_zombies/modinfo.json:9]:
```suggestion
    "category": "total_conversion",
```

**Pull request merged**